lib/credstashApi.js

```diff
@@ -7,8 +7,8 @@ const COMMAND = 'credstash';
 const CREDSTASH_ERROR = 'Error fetching configuration from CredStash. Error message:';
 
 /**
- * Creates a new CredstashApi object and executes the commands using node's child_process.spwan.
- * May receive an options objcet, that may contain the following attributes:
+ * Creates a new CredstashApi object and executes the commands using node's child_process.spawn.
+ * May receive an options object, that may contain the following attributes:
  * table - the table name on DynamoDB, default is credential-store.
  * region - AWS region.
  * context - CredStash context parameters.
```

Comment:
The implementation of the 'r' flag in the CredstashApi constructor seems to be wrong (according to the Credstash docs it should be '-r', I also did a check that showed that after changing it to '-r' Credstash takes it into consideration unlike with the 'r' implementation).

I didn't change it in order to not impact our apps that currently use the cApi object but I can do it if it's wanted.
It will also require changing the CredstashCli in order to support choosing different AWS regions to run the commands against.